spec/features/chitter_spec.rb

```diff
@@ -0,0 +1,52 @@
+require './app.rb'
+require 'pg'
+
+feature 'display' do
+  connection = PG.connect(dbname: 'chitter_test')
+
+  scenario 'can display the main Chitter board' do
+    visit '/'
+    expect(page).to have_css('h2', text: "Peep Board")
+  end
+
+  scenario 'can display a peep on the board' do
+    connection.exec("insert into peeps (name, content, time_stamp) values
+    ('Name1', 'Content1', NOW());")
```

Comment:
I like your use of the SQL function `Now()`. This can also be achieved by setting a default value on your time_stamp column when creating the table